CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(naoqi_driver)
 
+add_definitions(-std=gnu++11)
```

Comment:
@vrabaud I know the difference between `c++11` and `gnu++11` :-) my point was that `libqi` uses `gnu++11` for good reasons, I assume, and I wanted to ensure the code which depends on `libqi` has access to the same extensions. But you are right, if the code compiles with `c++11` then it is definitely more portable. I'll check and update the PR accordingly.